src/get-started/learn-more.md

```diff
@@ -1,54 +1,163 @@
 ---
-title: Learn more
-description: More resources to help you learn Flutter.
+title: Learn
+description: Resources to help you learn Flutter.
 prev:
   title: Write your first Flutter app
   path: /get-started/codelab
 ---
 
-Learn more about the Flutter framework from the following pages:
-
-## Flutter basics
-
-* [Introduction to Flutter's UI][]
-* [Building layouts][] tutorial
-* [Add interactivity][] tutorial
-
-## Apply your existing knowledge
-
-* [Flutter for Android developers][]
-* [Flutter for SwiftUI developers][]
-* [Flutter for UIKit developers][]
-* [Flutter for React Native developers][]
-* [Flutter for web developers][]
-* [Flutter for Xamarin.Forms developers][]
-
-## Other resources
-
-* [Flutter samples][]
-* [Flutter cookbook][]
-* [Bootstrap into Dart: learn more about the language][]
-* [Flutter API Docs][]
-* [The Complete Flutter Developer Bootcamp Using Dart][]
-* [Udacity online Flutter training][]
-
-Reach out to us at our [mailing list][]. We'd love to hear from you!
-
-Happy Fluttering!
-
-[Add interactivity]: {{site.url}}/ui/interactivity
-[Bootstrap into Dart: learn more about the language]: {{site.url}}/resources/bootstrap-into-dart
-[Building layouts]: {{site.url}}/ui/layout/tutorial
-[The Complete Flutter Developer Bootcamp Using Dart]: https://www.appbrewery.co/p/flutter-development-bootcamp-with-dart
-[Flutter API Docs]: {{site.api}}
-[Flutter cookbook]: {{site.url}}/cookbook
-[Flutter for Android developers]: {{site.url}}/get-started/flutter-for/android-devs
-[Flutter for SwiftUI developers]: {{site.url}}/get-started/flutter-for/swiftui-devs
-[Flutter for UIKit developers]: {{site.url}}/get-started/flutter-for/uikit-devs
-[Flutter for React Native developers]: {{site.url}}/get-started/flutter-for/react-native-devs
-[Flutter samples]: https://flutter.github.io/samples
-[Flutter for web developers]: {{site.url}}/get-started/flutter-for/web-devs
-[Flutter for Xamarin.Forms developers]: {{site.url}}/get-started/flutter-for/xamarin-forms-devs
-[Introduction to Flutter's UI]: {{site.url}}/ui
-[mailing list]: mailto:{{site.email}}
-[Udacity online Flutter training]: https://www.udacity.com/course/build-native-mobile-apps-with-flutter--ud905
+## Layout
+
+Given that Flutter is a UI toolkit, you’ll spend a lot of time creating layouts
+with Flutter widgets. In this section, you’ll learn how to build simple and
+complex layouts in Flutter with some of the most common layout widgets in
+Flutter. You’ll also learn about how to use DevTools to understand how your apps
+are being laid out, and you’ll learn how to debug one of the most common Flutter
+errors, which is related to the most common Flutter layout widgets.
+
+### Introduction to layouts
+
+The following tutorial will introduce you to layouts in Flutter, and give you an
+opportunity to work with the most common layout widgets.
+
+1. <i class="material-icons" aria-hidden="true">bolt</i> Take action! - Tutorial: [Layouts in Flutter][]
```

Comment:
(As Khanh pointed out in the original doc) We should have an icon of some sort that says "Take Action!" next to resources that allow/encourage the user to write code. 

This will obviously need to be styled and ideally made into a Liquid variable a la {{site.alert...

Reply:
@sfshaza2, you use icons next to links sometimes, right? Is there a set/font that you pull from and we could reuse here?

Reply:
FWIW, I'm using `material-symbols`, which I think is the icon font that the rest of the site uses. (I think it's the only icon font loaded into the site)